docs/learn/learn-nominator.md

```diff
@@ -48,15 +48,43 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen.md#optimizations) for more details.
 
+### Staking Election Stages
+
+The staking election system has 3 stages for both validators and nominators, namely "intention", 
+"electable/electing", and "active".
+
+- **intention to nominate:** an account that has stated the intention to nominate; also called simply 
+a "nominator".
+- **electing nominator:** a nominator who is selected to be a part of the input to the NPoS election 
+algorithm. This selection is based on stake, and is done using the bags-list pallet.
+- **active nominator:** a nominator who came out of the NPoS election algorithm backing an active validator, 
+sharing their rewards and slashes.
+
+![Nominator Election](../assets/staking/nominator-election.png)
+
+
 ### Required Minimum Stake
 
 Due to the way the [Phragmen algorithm](learn-phragmen.md) generates the solution set, and due to
-the fact that the solution set must fit in a single block, in some eras, a minimum number of DOT
-will be required to nominate with in order to receive staking rewards.
+the fact that the solution set must fit in a single block, a minimum number of DOT will be required to 
+nominate with, in order to receive staking rewards, can change between the eras.
+
+- **min-intention-threshold:** minimum stake to declare the intention to nominate. This parameter can be 
+updated via on-chain governance and the most recent and up to date version can be found on 
+[chain state](https://polkadot.js.org/apps/#/chainstate) (select **state query > staking > minimumNominatorBond**)
+
+- **min-electing:** minimum stake among the electing nominators. Since this is almost always the same as 
+“min-active”, it might not be reported
```

Comment:
```suggestion
- **min-electing:** minimum stake among the electing nominators. Since this is almost always the same as 
“min-active”, it might not be reported.
```